apps/odoo/docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.9"
+
+services:
+  odoodb:
+    container_name: odoodb
+    image: postgres:15
+    user: root
+    environment:
+      - POSTGRES_USER=odoo
+      - POSTGRES_PASSWORD=odoo17@2023      
```

Comment:
This password should be randomized using an [env variable](https://runtipi.io/docs/contributing/adding-a-new-app#user-defined-environment-variables).
[Here](https://github.com/runtipi/runtipi-appstore/blob/a61a33402037e31905d0c86c9fe110828bdbdda8/apps/activepieces/config.json#L22) is an example on how it was done in another app.